src/components/listItemComponent/ListItemComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import { useSelector } from "react-redux";
+import { selectRedditReturn } from '../reddit/RedditSlice'
+
+const ListItemComponent = (props) => {
+    const redditApiReturn = useSelector(selectRedditReturn);
+    console.log('redditApiReturn', redditApiReturn)
+
+    if(!redditApiReturn.data) {
```

Comment:
this will return truthy is data is an empty object just fyi. this is probably ok to see if the api has returned though

Reply:
<img width="360" alt="image" src="https://user-images.githubusercontent.com/74274857/160217127-ec45e161-5c06-4f44-8d6b-c20b285a4d3c.png">
tested,  this is undefined = false before the api fetches()

Reply:
yes i know that's why it's ok for checking for api response, but not ok for checking for empty object in case it returned empty. just telling you so you remember that empty objects are truthy